src/Stratis.Bitcoin/Base/BaseFeature.cs

```diff
@@ -302,6 +303,7 @@ public static IFullNodeBuilder UseBaseFeature(this IFullNodeBuilder fullNodeBuil
                     services.AddSingleton<INetworkPeerFactory, NetworkPeerFactory>();
                     services.AddSingleton<NetworkPeerConnectionParameters>(new NetworkPeerConnectionParameters());
                     services.AddSingleton<IConnectionManager, ConnectionManager>();
+                    services.AddSingleton<ConnectionManagerSettings>(new ConnectionManagerSettings());
```

Comment:
No need to create the instance here
Just do 
```
  services.AddSingleton<ConnectionManagerSettings>()
```

Reply:
+1

Reply:
Done.